lib/sdn_epc/converter.ex

```diff
@@ -0,0 +1,107 @@
+defmodule SdnEpc.Converter do
+  require SdnEpc.OfpmRecord
+
+  @ofp_version Application.get_env(:sdn_epc, :ofp_version)
+
+  def convert({:packet_in, xid, body}) do
```

Comment:
It could be that way. What I suggested was more like:

```elixir
def convert(msg) do
   SdnEpc.OfpmRecord.ofp_message(
       version: @ofp_version,
       xid: xid,
       body: body(msg))
end

defp body({:packet_in, ..}), do: ...
defp body({:ofp_feature_request, ...}), do: ...
```